ovirtlib4/templates.py

```diff
@@ -21,7 +21,7 @@ def _get_collection_entity(self):
         return TemplateEntity(connection=self.connection)
 
 
-class TemplateEntity(CollectionEntity):
+class TemplateEntity(CollectionEntity, ClusterEntity):
```

Comment:
Again - please share with me why you consider inheritance to be suitable here?
By using inheritance you are saying that a template is a cluster with added features.
If you would use aggregation you would be saying - 'this template resides within a cluster that exists without being dependant of the template

Reply:
It means Template have Cluster href property, meaning template is associate with a cluster